net/WebSocketHandler.hpp

```diff
@@ -45,7 +45,7 @@ class WebSocketHandler : public ProtocolHandlerInterface
     const bool _isClient;
 
     // Last member.
-#ifdef ENABLE_DEBUG
```

Comment:
Note: #ifdef ENABLE_DEBUG is always true as config.in will always define it. In a production build config.in contains the following:

#define ENABLE_DEBUG 0

Since _unit was always instantiated in Online builds, production or not, I removed the #ifdef ENABLE_DEBUG.

Reply:
great catch =)